test/e2e/README.md

```diff
@@ -0,0 +1,15 @@
+# End-to-end tests
+## Executing tests
+### Using ginkgo locally
+From the velero directory:
+hack/tools/bin/ginkgo test/e2e
```

Comment:
I don't see this `tools` directory. Is it going to be added later?

Reply:
If I run `go get github.com/onsi/ginkgo/ginkgo`, I can then run `ginkgo test/e2e` successfully. So I think the docs probably just need to be updated.

Reply:
Actually:

```shell
cd test/e2e
make build
```

Then you'll have the ginkgo that this expects.

Reply:
@dsu-igeek I think you did not pull the change I pushed.

Reply:
I think my commits got dropped with this merge commit `513176b21a5b8a84b7a61edeb16cddbe3cd6a247`

Reply:
We removed all of the build steps for ginkgo and it is now installed to and run from the $GOPATH/bin